### PR TITLE
Stepan/tg-138

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: webapp/src/component/navigation/NavigationWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <Box
+      style={{
+        borderBottom: `1px solid ${grey[200]}`,
+        backgroundColor: grey[50],
+      }}
+      pl={4}
+      pr={4}
+    >
+      <div className={classes.container}>{children}</div>
```
Comment (JanCizmar) [inline webapp/src/component/navigation/NavigationWrapper.tsx:27]: Do we really need Box and div?

File: webapp/src/component/navigation/NavigationWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import grey from '@material-ui/core/colors/grey';
+import { Box, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  container: {
```
Comment (JanCizmar) [inline webapp/src/component/navigation/NavigationWrapper.tsx:6]: display, alignItems could be provided as props of Box component

File: webapp/src/component/navigation/NavigationWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    height: 48,
+    display: 'flex',
+    alignItems: 'center',
+    '& > * + *': {
+      marginLeft: 10,
```
Comment (JanCizmar) [inline webapp/src/component/navigation/NavigationWrapper.tsx:11]: Do you really need exact 10px? Component Box has "ml" or "marginLeft" prop or you can use theme.spacing function.

File: webapp/src/component/layout/DashboardPage.tsx [hunk 4]
```diff
@@ -1,90 +1,40 @@
...
 
 export const DashboardPage: FunctionComponent<DashboardPageProps> = ({
   children,
-  sideMenuItems,
   ...props
 }) => {
   const classes = useStyles({});
 
   return (
     <div className={classes.root}>
```
Comment (JanCizmar) [inline webapp/src/component/layout/DashboardPage.tsx:34]: Using <Box> you don't have to use that many style props.

File: webapp/src/component/navigation/NavigationWrapper.tsx (hunk 1 above)
Comment (stepan662) [reply webapp/src/component/navigation/NavigationWrapper.tsx]: Fixed

File: webapp/src/component/navigation/NavigationWrapper.tsx (hunk 2 above)
Comment (stepan662) [reply webapp/src/component/navigation/NavigationWrapper.tsx:6]: Fixed

**Pull request merged**